genome_designer/main/xhr_handlers.py

```diff
@@ -96,12 +97,15 @@ def create_ref_genome_from_browser_upload(request):
 
     uploaded_file = request.FILES['refGenomeFile']
 
+    # Substitute non-alphanumeric characters in filename with underscores
+    sanitized_file_name = re.sub('[\W]', '_', uploaded_file.name)
```

Comment:
This code should be in a more central place that other code paths that import refernece genomes use.

In particular, `create_ref_genome_from_server_location` should use this.  And, while in theory it should be safe, might as well have `create_ref_genome_from_ncbi` use this as well.

It looks like all these functions ultimately go through `import_reference_genome_from_local_file` so that's probably where you want this code.